index.html

```diff
@@ -0,0 +1,1376 @@
+                            FEATURES
+                            <span class="mg-section ">
+                                Health
+                            </span>
+                        </span>
+
+                        <p class="mg-p section-heading font-weight-bold mt-2">
+                            <a href="#" class="feature-hover">Doctors Are Turning to Artificial Intelligence to Diagnose
+                                Cancer Patients</a>
+                        </p>
+                    </div>
+                </article>
+                <article class="col-md-3 pr-0 mr-0">
+                    <div class="magazine-img-wrapper">
+                        <img src="assets/magazine-4.jpg" alt="magazine" class="mt-3 mb-2 mg-img-4 img-fluid">
+                        <span class="mg-category font-weight-bold section-heading mb-0 pr-0">
+                            features
+                            <span class="mg-section ">
+                                Health
+                            </span>
+                        </span>
+
+                        <p class="mg-p section-heading font-weight-bold mt-2">
+                            <a href="#" class="feature-hover">Precision Medicine Is Crushing Once-Untreatable
+                                Cancers</a>
+                        </p>
+                    </div>
+                </article>
+            </section>
+            <!--end of In magazine section Part 1-->
+        </div>
+        <!--end of In magazine section Part 1-->
+
+
+
+        <div class="container-fluid">
+            <!--Start of Magazine section 2-->
+            <section class="row">
+                <article class="col-md-3">
+                    <div class="magazine-img-wrapper">
+                        <img src="assets/magazine-5.jpg" alt="magazine" class="mt-3 mb-2 mg-img-5 img-fluid">
+                        <span class="mg-category font-weight-bold section-heading mb-0 pr-0">
+                            DOWNLOADS
+                            <span class="mg-section ">
+                                culture
+                            </span>
+                        </span>
+
+                        <p class="mg-p section-heading font-weight-bold mt-2">
+                            <a href="#" class="feature-hover">How Harvard Caved to Students and Fired Its First Black
+                                Faculty Dean</a>
+                        </p>
+                    </div>
+
+                </article>
+
+                <article class="col-md-3 pr-0 mr-0">
+                    <div class="magazine-img-wrapper">
+                        <img src="assets/magazine-6.jpg" alt="magazine" class="mg-img-8 mt-3 mb-2  mg-img-5 img-fluid">
+                        <span class="mg-category font-weight-bold section-heading mb-0 pr-0">
+                            Downtime
+                            <span class="mg-section ">
```

Comment:
The same semantic tags `<section>` and `<article>` apply to EDITOR PICKS section.